### ISSUE TITLE
A11y_.NET Core_WinForms_ToolStripContainer_Bottomtool bar_AI4D: Incorrect Name property is provided to "Toolstipbutton1" & "ToolstipSplitButton1" buttons.

### ISSUE THREAD

**Issue by HindujaArugula** [dotnet/winforms#9273]
### .NET version

.NET-SDK-8.0.100-preview.4.23260.5

### Did it work in .NET Framework?

No

### Did it work in any of the earlier releases of .NET Core or .NET 5+?

No response

### GitHub Tags:
#A11yMAS;#A11yTCS;#BM_.NET Core[Winforms]-Win32-May2023;#.NET Core;#A11ySev3;#WCAG4.1.2#DesktopApp;#Win11;#NonBenchmark;#AI4D;

### Actual:
Incorrect Name property is provided to "ToolStripbutton1" & "ToolStripSplitButton1" buttons.

### Expected:
Name property should be provided correctly, and the word "Button" should be removed from "ToolStripbutton1" & "ToolStripSplitButton1" buttons name should be provided as "ToolStrip" & "ToolStripSpilt1".

### User Impact:
Screen reader only user will get confused about the "ToolStripbutton1" & "ToolStripSplitButton1" if word button is provided in the name.

### Steps to reproduce

1. Install and Open "Winforms" App.
2. "Main Form" screen would appear.
3. TAB to "MenuToolbars_Controls:Testing the controls under the Menu Toolbars tab" and press ENTER key.
4. "Main Form" screen appears.
5. TAB to "ToolStripContainer" button and press ENTER key.
6. "ToolStripContainer" screen appears.
7. Open Accessibility Insights.
8. Now TAB to Bottom section and observe that Incorrect name property is provided to "ToolStripbutton1" & "ToolStripSplitButton1" buttons.

### Attachment:
![AI4D](https://github.com/dotnet/winforms/assets/125462007/ddd2ea0f-f583-4c69-a7b1-e6f0b847edfb)

**Comment by merriemcgaw** [comment]
This is Sev3 and has no SLA for getting it fixed. @Olina-Zhang  if your team could adjust the names in the sample app so they don't flag this next time, that would be awesome. You can then close this issue.

**Comment by Olina-Zhang** [comment]
@merriemcgaw, @HindujaArugula  I have updated the testing application: I think we should keep the button and splitButton to preserve their properties and distinguish them. Just added a hyphen for these control items.

![Issue9273](https://github.com/dotnet/winforms/assets/26474449/524de7d3-87c8-4f6e-b6c1-6a5387fdaf8f)

**Comment by MelonWang1** [comment]
Verified this issue in the latest .NET 8.0 SDK build: 8.0.100-preview.7.23323.19 on winforms repo, it was fixed. ToolStripButton1 and toolStripSplitButton1 name property has changed. Screenshot as below:
![Screenshot 2023-06-24 230650](https://github.com/dotnet/winforms/assets/94418985/02dddb28-06a6-4c99-918f-9963d21f045f)
![Screenshot 2023-06-24 230743](https://github.com/dotnet/winforms/assets/94418985/210afccb-bd9d-4b96-90cd-601bd96ee436)

**Comment by MandiMan** [comment]
Verified this on .NET 8.0 TP build: 8.0.100-preview.6.23327.6, issue was fixed, result is same as above test result.
